episodes/continuous-integration.md

```diff
@@ -94,6 +94,16 @@ It also enables the developer to run unit testing in an automated way to discove
 CICD relies on version control.
 Workflows are usually triggered by events, such as merging to the main branch happening on the verson control repository.
 
+# Docker Containers
+
+Nowadays software is often distributed or deployed using containers.
+These containers contain every dependency an application needs and can run anywhere.
+This is especially useful in applications that run as a service on a cloud provider such as Google or Amazon Web Services where it is not known before hand where an application will run.
+These containers are light weight operating system images in which the application is stored including everything it needs.
+To build these container images, CI/CD is extremely useful as it can be automated as explained below in the section explaining pipelines and workflows.
+Running applications in containers tends to enforce decoupling from external dependencies and communicate to external services through well defined and stable interfaces.
+Building and distributing containers is generally done using [docker](https://www.docker.com) but there are others such as [podman](https://podman.io/)
+
```

Comment:
```suggestion
You can find more information about containers [here](https://book.the-turing-way.org/reproducible-research/renv/renv-containers.html).
```
Turing Way has a fairly detailed chapter on containers (using docker, podman and singularity). Maybe we can point to it as further reading.